client/src/components/addEventComponent.js

```diff
@@ -0,0 +1,250 @@
+import React, { Component } from 'react';
```

Comment:
Most of the feedback for `/addCenterComponent.js` applies here too. I also see they use similar functions. You should consider extracting these functions and calling them where needed to keep things DRY.